package.json

```diff
@@ -58,7 +59,7 @@
     "pretty-one": "prettier --write \"$1\".js",
     "pretty:check": "prettier --check .",
     "proxy": "prism proxy --errors $npm_package_config_specfile https://api.lob.com/v1",
-    "redoc": "redoc-cli bundle $npm_package_config_specfile -t actions/redoc/template.hbs --options.hideDownloadButton=true --options.expandResponses='200,201' --templateOptions.segmentKey=0ysbW8vXHeRAnY2saZPMlRSZFfPwd1RO --templateOptions.SUPPORT_URL=https://support.lob.com/hc/en-us --templateOptions.LOB_ASSETS_URL=https://s3-us-west-2.amazonaws.com/public.lob.com --templateOptions.canonicalUrl=https://docs.lob.com/ --templateOptions.keywords=\"Direct Mail, Address Verification\" --templateOptions.description=\"Detailed developer documentation for Lob's address verification API and print & mail API. Learn how to easily integrate Lob's APIs within your product or technology stack.\" --templateOptions.LOB_URL=https://lob.com --templateOptions.image=https://s3-us-west-2.amazonaws.com/public.lob.com/marketing/opengraph2.png --templateOptions.DASHBOARD_URL=https://dashboard.lob.com/# --title=\"DRAFT NEW Lob API documentation\" --options.noAutoAuth=true --options.maxDisplayedEnumValues=15 --options.theme.colors.primary.main=#27333c --options.theme.sidebar.backgroundColor=#f7f9fa --options.jsonSampleExpandLevel=1 --options.requiredPropsFirst=true --options.theme.typography.code.backgroundColor=#ebf0f6 --options.theme.typography.code.fontFamily=JetBrainsMono --options.theme.typography.code.color=#27333c --options.theme.typography.fontFamily=\"Larsseit,sans-serif;\" --options.theme.typography.links.color=#0099d7 --options.theme.typography.links.textDecoration=none --options.theme.codeBlock.backgroundColor=#ecf0f5 --options.theme.rightPanel.backgroundColor=#f7f9fa --options.theme.rightPanel.textColor=#416581 --options.theme.typography.fontSize=16 -o docs/index.html",
+    "redoc": "redoc-cli bundle $npm_package_config_specfile -t actions/redoc/template.hbs --options=docs/theme.json --templateOptions=docs/template.json --title=\"Lob API documentation\" -o docs/index.html",
```

Comment:
![image](https://user-images.githubusercontent.com/45194378/130303456-5194743c-4877-4799-9346-6758c426d74d.png)
don't think separating it is working -sweat-

Reply:
you might have to write a patch for this in `redoc-cli` cuz their `templateOptions` was broken to begin with